### PR TITLE
Bad input

### PR DESCRIPTION
**Pull request by Cafferoq**
### Summary

Input checking on client side. 

### Checklist

- [x] Did you run `npm test`?
- [x] Did you update/add documentation for new methods or changed functionality?

Finally, is it ready to be reviewed and merged: yes :white_check_mark:

### How to Review

1. Pull down this branch and the accompanying server-side branch. 
2. Run them both.
3. Ensure that the admin override modal, the omnibar, and the checkout input disallow non-alphanumeric characters.

### Links

Put links to Jira tickets, and anything else, here.

- [TFF-90](https://msoese.atlassian.net/browse/TFF-90)
- [Sister PR](https://github.com/TheFourFifths/consus/pull/25)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "consus-client",
   "description": "The client application for Consus, an inventory management system",
-  "version": "0.1.0",
+  "version": "0.2.0",
```
Comment (tondef001) [inline package.json]: Not worthy of not approving, but should the version number really be changed in this PR?

Comment (thofmann) [inline package.json]: This is coming from changes in dev that made it into this task branch, but not the story branch.

I've merged dev into the story, and this diff should go away after I resolve a merge conflict here...

File: src/views/components/omnibar.jsx [hunk 2]
```diff
@@ -1,7 +1,8 @@
 import React from 'react';
 
 import { Link } from 'react-router';
-import { searchStudent } from '../../lib/api-client'
+import { searchStudent } from '../../lib/api-client';
```
Comment (thofmann) [inline src/views/components/omnibar.jsx]: :+1:

File: src/views/components/input-modal.jsx [hunk 3]
```diff
@@ -32,10 +46,12 @@ export default class InputModal extends React.Component {
                 onClose={this.onAccept.bind(this)}>
                 <p>{this.props.message}</p><br/>
                 <input
+                    maxLength="30"
```
Comment (thofmann) [inline src/views/components/input-modal.jsx]: :+1:

File: src/views/components/input-modal.jsx [hunk 4]
```diff
@@ -16,14 +17,27 @@ export default class InputModal extends React.Component {
...
     }
 
     onCancel(){
         this.setState({input:''});
         this.props.onCancel();
     }
 
+    renderWarning(){
+        if(this.state.invalid){
+            return <span className="invalidText">Please only use Alphanumeric characters.<br/></span>
```
Comment (thofmann) [inline src/views/components/input-modal.jsx]: Missing `;`

File: src/views/components/input-modal.jsx [hunk 5]
```diff
@@ -6,7 +6,8 @@ export default class InputModal extends React.Component {
     constructor(props){
         super(props);
         this.state = {
-            input: ''
+            input: '',
+            invalid: false
         }
```
Comment (thofmann) [inline src/views/components/input-modal.jsx]: Not a problem created in this PR, but missing a `;` here.

**Pull request merged**